package-lock.json

```diff
@@ -1,6000 +0,0 @@
-{
```

Comment:
we don't want to update the packages here. use `npm ci` instead that will install the packages from the lock file and doesn't generate a new one.

Reply:
I have used it yet.